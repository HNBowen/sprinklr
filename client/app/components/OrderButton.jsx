import React from 'react'

const OrderButton = (props) => {

}

export default OrderButton