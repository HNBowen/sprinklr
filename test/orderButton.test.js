import React from 'react';
import {shallow} from 'enzyme'
import {shallowToJson} from 'enzyme-to-json'

import OrderButton from '../client/app/components/OrderButton.jsx'

describe('OrderButton', function() {

  it('should render correctly', function() {
    const wrapper = shallow(<OrderButton />)
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  })
})