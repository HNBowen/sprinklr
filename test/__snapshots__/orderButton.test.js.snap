// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderButton should render correctly 1`] = `""`;
